src/App.jsx

```diff
@@ -69,7 +62,7 @@ export function App() {
 							listToken ? (
 								<Navigate to="/list" />
```

Comment:
A little thing...should we switch our React Router Dom links to single quotes for consistency?